### PR TITLE
Polishing issues discovered by FindBug

### PR DESCRIPTION
**Pull request by pjanouse**

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/proccleaner/PreBuildCleanup.java [hunk 1]
```diff
@@ -69,6 +68,7 @@ public Environment setUp(AbstractBuild build, Launcher launcher,
     }
 
     @Extension
+    @SuppressWarnings("deprecation")
```
Comment (olivergondza) [inline src/main/java/org/jenkinsci/plugins/proccleaner/PreBuildCleanup.java]: Can it be moved to more focused scope than the whole class? Not sure what is the deprecated API here...

File: src/main/java/org/jenkinsci/plugins/proccleaner/PsProcess.java [hunk 2]
```diff
@@ -116,8 +117,11 @@ public boolean equals(Object o) {
                 return true;
         return false;
     }
-    
-    //TODO implement hashCode etc.
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(pid, ppid, args, ptree);
```
Comment (olivergondza) [inline src/main/java/org/jenkinsci/plugins/proccleaner/PsProcess.java]: Careful the `hashCode` is out of sync with `equals`

Comment (pjanouse) [reply src/main/java/org/jenkinsci/plugins/proccleaner/PsProcess.java]: Good spot! I'd change ```hashCode()``` following ```equals()``` impl. although I'm not happy with the original...

**Review by olivergondza — APPROVED** [review]

**Review by vjuranek — APPROVED** [review]

**Comment by vjuranek** [comment]
Merged, thanks!

**Pull request merged**